spk/dotnet9-runtime/Makefile

```
@@ -0,0 +1,40 @@
+SPK_NAME = dotnet9-runtime
+SPK_VERS = 9.0.0
+SPK_REV = 1
+SPK_ICON = src/dotnet.png
+
+OPTIONAL_DEPENDS = 
+DEPENDS = cross/dotnet9-runtime
+
+REQUIRED_MIN_DSM = 7.0
+
+# Arch exclusions for dotnet
+DOTNET_CORE_ARCHS = 1
+
+MAINTAINER = brice.fromentin
```

this must match the github user name:
```suggestion
MAINTAINER = brice-fromentin
```